packages/sysutils/busybox/scripts/init

```diff
@@ -991,7 +993,7 @@
     FLASH_FREE=$(( $FLASH_FREE * 1024 ))
 
     # Disregard kernel size if it's a a block device, which is the case on Amlogic/WeTek devices
-    if [ ! -b $IMAGE_KERNEL ]; then
+    if [ ! -b /$IMAGE_KERNEL ]; then
```

Comment:
Can **$IMAGE_KERNEL** has spaces in it? Then it should be in double quotes as everywhere else.

Reply:
Unlikely (it would be `/dev/boot`, normally) but I've added double quotes where they were not used - thanks!